homeassistant/components/anglian_water/coordinator.py

```diff
@@ -0,0 +1,60 @@
+"""DataUpdateCoordinator for anglian_water."""
+
+from __future__ import annotations
+
+from datetime import timedelta
+
+from pyanglianwater import AnglianWater
+from pyanglianwater.exceptions import (
+    ExpiredAccessTokenError,
+    InvalidPasswordError,
+    InvalidUsernameError,
+    ServiceUnavailableError,
+    UnknownEndpointError,
+)
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN, LOGGER
+
+
+class AnglianWaterDataUpdateCoordinator(DataUpdateCoordinator):
```

Comment:
Please add the return type of `.data`

```suggestion
class AnglianWaterDataUpdateCoordinator(DataUpdateCoordinator[..]):
```